src/utilities/messagesHelper.js

```diff
@@ -0,0 +1,62 @@
+import store from '../redux/store';
+import i18n from '../translations/i18n';
+import { NotificationActions } from '../redux/actions/notificationActions';
+
+export function webError(response) {
+
+  function error400(error) {
+    // Validation problem
+    if (error.title === 'One or more validation errors occurred.') {
+      console.log(error.errors);
+      store.dispatch(NotificationActions.error(i18n.t('error.validationError')));
+      return;
+    }
+  
+    // Other problems
+    if (error.errors.message[0]) {
+      store.dispatch(NotificationActions.error(error.errors.message[0]));
```

Comment:
```suggestion
      error.errors.message[0] && store.dispatch(NotificationActions.error(error.errors.message[0]));
```

Reply:
There is no need for doing it. The error will be thrown on the upper level and a generic message will be displayed to the user.